models/marts/core/int_all_graph_resources.sql

```diff
@@ -23,116 +23,116 @@ with unioned as (
+        unioned_with_calc.version as version,
+        unioned_with_calc.deprecation_date as deprecation_date,
+        unioned_with_calc.is_contract_enforced as is_contract_enforced,
+        unioned_with_calc.total_defined_columns as total_defined_columns,
+        unioned_with_calc.total_described_columns as total_described_columns,
+        unioned_with_calc.exposure_type as exposure_type,
+        unioned_with_calc.maturity as maturity,
+        unioned_with_calc.url as url,
+        unioned_with_calc.owner_name as owner_name,
+        unioned_with_calc.owner_email as owner_email,
+        unioned_with_calc.meta as meta,
+        unioned_with_calc.macro_dependencies as macro_dependencies,
+        unioned_with_calc.metric_type as metric_type,
+        unioned_with_calc.label as label,
+        unioned_with_calc.metric_filter as metric_filter,
+        unioned_with_calc.metric_measure as metric_measure,
+        unioned_with_calc.metric_measure_alias as metric_measure_alias,
+        unioned_with_calc.numerator as numerator,
+        unioned_with_calc.denominator as denominator,
+        unioned_with_calc.expr as expr,
+        unioned_with_calc.metric_window as metric_window,
+        unioned_with_calc.grain_to_date as grain_to_date,
         unioned_with_calc.source_name, -- NULL for non-source resources
-        unioned_with_calc.is_source_described, 
-        unioned_with_calc.loaded_at_field, 
-        unioned_with_calc.loader, 
-        unioned_with_calc.identifier,
-        unioned_with_calc.hard_coded_references, -- NULL for non-model resources
-        unioned_with_calc.number_lines, -- NULL for non-model resources
-        unioned_with_calc.sql_complexity, -- NULL for non-model resources
-        unioned_with_calc.is_excluded -- NULL for metrics and exposures
+        unioned_with_calc.is_source_described as is_source_described,
+        unioned_with_calc.loaded_at_field as loaded_at_field,
+        unioned_with_calc.loader as loader,
+        unioned_with_calc.identifier as identifier,
+        unioned_with_calc.hard_coded_references as hard_coded_references, -- NULL for non-model resources
+        unioned_with_calc.number_lines as number_lines, -- NULL for non-model resources
+        unioned_with_calc.sql_complexity as sql_complexity, -- NULL for non-model resources
+        unioned_with_calc.is_excluded as is_excluded -- NULL for metrics and exposures
 
     from unioned_with_calc
     left join naming_convention_prefixes
         on unioned_with_calc.prefix = naming_convention_prefixes.prefix_value
 
-    cross join naming_convention_folders   
+    cross join naming_convention_folders
 
-), 
+),
 
 calculate_model_type as (
-    select 
-        *, 
-        case 
-            when resource_type in ('test', 'source', 'metric', 'exposure', 'seed') then null
+    select
+        *,
+        case
+            when joined.resource_type in ('test', 'source', 'metric', 'exposure', 'seed') then null
```

Comment:
This is not the convention of this project, we don't qualify the name of tables/CTEs when we read just from one. Could you revert back to not qualifying the table/CTE name?

Reply:
removed qualified names where there is no need using them